setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+VERSION = 1
+
+setup(
+    name='atool',
+    version=VERSION,
+    packages=find_packages(),
+    include_package_data=True,
```

Comment:
I would suggest using [`package_data`](https://packaging.python.org/en/latest/distributing.html#package-data) so you don't include extraneous files like those in the `debian` directory. You can write and use a function.